samples/apps/txregulator/tests/txregulatorclient.py

```diff
@@ -269,9 +264,8 @@ def run(args):
                     if tx_id not in revealed_tx_ids:
                         check(
                             c.rpc("REG_get_revealed", {"tx_id": tx_id}),
-                            error=lambda e: e is not None
-                            and e["code"]
-                            == infra.jsonrpc.ErrorCode.INVALID_PARAMS.value,
+                            error=lambda status, msg: status
+                            == http.HTTPStatus.BAD_REQUEST.value,
```

Comment:
There's enough of those that it might be tempting to

```
def returns(rc):
  return lambda status, _msg: status == rc.value
...
...
error = returns(http.HTTPStatus.BAD_REQUEST)
```

Reply:
Nice, much cleaner.